### DISCUSSION TITLE
ant-design Form真的有支持chrome吗，官网都是坏的啊

### DISCUSSION THREAD

**Discussion by jackhhaahah** [ant-design/ant-design#45062] [General]
<img width="1403" alt="image" src="https://github.com/ant-design/ant-design/assets/110645104/7b64d0a8-de23-4fb5-a54e-b440acdf4cde">
提交按钮按第一下，错误提示的位置就会如图，被挡住，点第二下，才会变正常显示

到safari就没问题。

Chrome Version 116.0.5845.187

**Comment by yoyo837** [comment]
chunk 加载失败了吧? 看看网络控制台

**Reply by yoyo837** [reply]
或用国内镜像试一试

**Reply by jackhhaahah** [reply]
<img width="1717" alt="image" src="https://github.com/ant-design/ant-design/assets/110645104/eca966e6-598c-4c86-9577-8a565360ef61">
没看到什么js加载失败～，
用国内的镜像也一样https://ant-design.gitee.io/components/form-cn
你的chrome不会这样吗

**Reply by yoyo837** [reply]
看不到你描述的问题

**Reply by jackhhaahah** [reply]
有个extension乱加css导致的，抱歉打扰了，ant-design很棒(^v^)

**Reply by afc163** [reply]
哪个 extension 捣乱，发一下示众！
